schema/field/annotation.go

```diff
@@ -19,6 +19,30 @@ type Annotation struct {
 	//	}
 	//
 	StructTag map[string]string
+
+	// ID defines a multi-field schema identifier. Note,
+	// the annotation is valid only for edge schemas.
+	//
+	//	func (TweetLike) Annotations() []schema.Annotation {
+	//		return []schema.Annotation{
+	//			field.ID("user_id", "tweet_id"),
+	//		}
+	//	}
+	//
+	ID []string
```

Comment:
This field can be named as `CompositeID` or `IDs`

Reply:
See my comment above.